defaults/main.yml

```diff
@@ -35,6 +34,10 @@ pyenv_debian_packages:
   - libreadline-dev
   - libsqlite3-dev
   - libssl-dev
+  - libxml2-dev
+  - libxmlsec1-dev
+  - llvm
```

Comment:
according to their doc `llvm` is only needed for `PyPy`, and even then I doubt it's really needed, as I was able to compile PyPy with GCC in the past. `build-essential` brings both `gcc` and `make`, so I suggest that `llvm` and `make` are removed from this list. They can be installed as separate steps outside of this role, if needed.

Reply:
Seems reasonable. Closing the pr